servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -0,0 +1,84 @@
+[[kiali-tutorial]]
+= Kiali tutorial
+
+This tutorial uses {ProductShortName} and the `bookinfo` tutorial to demonstrate how you can use the Kiali console to view the topography and health of your service mesh.
+
+Prerequisites:
+
+* OpenShift Container Catalog 3.10 or higher installed.
+* {ProductName} {ProductVersion} installed.
+* Kiali parameters specified in the custom resource file.
+* `Bookinfo` demonstration application installed.
+
+[[accessing-kiali-console]]
+== Accessing the Kiali console
+
+. A route to access the Kiali console should already exist.
++
+```
+$ oc get routes
+```
++
+While your exact environment may be different, you should see a result that's something like this:
++
+```
+NAME                   HOST/PORT                                                PATH      SERVICES               PORT              TERMINATION   WILDCARD
+grafana                grafana-istio-system.127.0.0.1.nip.io                          grafana                http                            None
+istio-ingress          istio-ingress-istio-system.127.0.0.1.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.127.0.0.1.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.127.0.0.1.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.127.0.0.1.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+```
++
+. Launch a browser and navigate to navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
++
+image:kiali-login.png[Login Page]
++
+
+
+== Graph page
+
+After you log in you should see the Graph page.  This page shows a graph with all the microservices, connected by the requests going through then. On this page, you can see how the services interact with each other.
+
+. From the Namespace menu, select bookinfo.  Now the graph only displays the services in the bookinfo application.
```

Comment:
\*Namespace\*
s/only displays/displays only
Check caps of bookinfo in all of these.